### DISCUSSION TITLE
could you recommend a good GUI to use with wgpu

### DISCUSSION THREAD

**Discussion by shi-yan** [gfx-rs/wgpu#5634] [Q&A] [answered]
I want to create a simple visualization tool using this library. I'm wondering what would be a good GUI library to use with wgpu?

Thanks.

**Comment by SludgePhD** [comment, marked as answer]
I've had a lot of success with [egui](https://github.com/emilk/egui) so far. It has Issues™ but gets the job done.

It even has a wgpu integration available already: https://github.com/emilk/egui/tree/master/crates/egui-wgpu
